developers/weaviate/api/graphql/additional-operators.md

```diff
@@ -526,24 +570,28 @@ For example:
 </Tabs>
 
 
-## group
+## `group` operator
+
+You can use a group operator to combine similar concepts (also known as _entity merging_). There are two ways of grouping semantically similar objects together, `closest` and `merge`. To return the closest concept, set `type: closest`. To combine similar entities into a single string, set `type: merge`
 
-You can use a group operator to combine similar concepts (aka _entity merging_). There are two ways of grouping objects with a semantic similarity together.
+The Python client v4 doesn't have a `group` operator. Use `graphql_raw_query` to issue a GraphQL query instead.
```

Comment:
Let's omit this line. I believe not having the group operator is not unique to the PyV4 client, and I think this is perhaps not the place to get into client-specific idiosyncrasies.